site/docs/components/password_input.mdx

```diff
@@ -0,0 +1,97 @@
+---
+name: Password input
+menu: Components
+route: /components/password-input
+---
+
+import { Playground } from "docz";
+import { StatusLabel } from "hds-react";
+
+import ColorBox from "../../src/components/ColorBox";
+import LargeParagraph from "../../src/components/LargeParagraph";
+import Link from "../../src/components/Link";
+import Text from "../../src/components/Text";
+
+# Password input
+
+<StatusLabel type="alert">Draft</StatusLabel>
+<StatusLabel type="success" style={{marginLeft: 'var(--spacing-xs)'}}>Accessible</StatusLabel>
+
+<LargeParagraph>
+    A password field is a foundational input field component that the user can interact with and specifically input password data.
+</LargeParagraph>
+
+## Principles
+- A label should always be provided with password fields.
+- If possible, allow the password input to reflect the strength of the password entered.
+- If possible, add other programmatic assistance. Detect and suggest changes to provide the user with assistance in generating strong passwords.
+- If possible, do input validation in client-side real-time and provide the user with immediate feedback. See HDS Form validation pattern page for more information and best practices with password fields.
+- Password fields should always be mandatory and it is recommended to mark it as a required field.
+- Use `autocomplete="new-password"` and `id="new-password"` for the password input in a sign-up form, and for the new password in a reset-password form.
+- Provide functionality to mask/unmask the password.
+- Use `aria-label` and `aria-describedby` for password inputs.
```

Comment:
@VilleMiekkoja @minevala Nämä taitavat olla sellaisia, jotka on jo otettu huomioon toteutuksessa itsessään? Tarvitseeko näitä erikseen mainita siis?

Reply:
Hmm.. this sentence feels incomplete or redundant 🤔